docs/tutorials/multinode/terraform/htcondor/noise.sub

```diff
@@ -0,0 +1,12 @@
+universe                = docker
```

Comment:
Which `noise.sub` and `noise.py` file(s) are required? We should clean up any that are no longer in use, and add comments if duplicates (e.g. `.../htcondor/noise.py` vs `.../multinode/noise3.py`) are necessary for certain workflows.